src/mono/wasm/debugger/DebuggerTestSuite/BreakpointTests.cs

```diff
@@ -219,23 +219,25 @@ public async Task ConditionalBreakpoint2(string function_to_call, string method_
         }
 
         [ConditionalTheory(nameof(RunningOnChrome))]
-        [InlineData("c == 15", 79, 3, 79, 11)]
-        [InlineData("c == 17", 79, 3, 80, 11)]
-        [InlineData("g == 17", 79, 3, 80, 11)]
-        [InlineData("true", 79, 3, 79, 11)]
-        [InlineData("\"false\"", 79, 3, 79, 11)]
-        [InlineData("\"true\"", 79, 3, 79, 11)]
-        [InlineData("5", 79, 3, 79, 11)]
-        [InlineData("p", 79, 3, 80, 11)]
-        [InlineData("0.0", 79, 3, 80, 11)]
-        public async Task JSConditionalBreakpoint(string condition, int line_bp, int column_bp, int line_expected, int column_expected)
+        [InlineData("c == 15", 79, 3, 79, 11, "dotnet.worker.js")]
+        [InlineData("c == 17", 79, 3, 80, 11, "debugger-driver.html")]
+        [InlineData("g == 17", 79, 3, 80, 11, "debugger-driver.html")]
+        [InlineData("true", 79, 3, 79, 11, "dotnet.worker.js")]
```

Comment:
What does the trace look like for this? Just wondering why this is in `dotnet.worker.js`.

Reply:
Thanks for the question, there was an issue, we are receiving the same scriptId, so we are replacing in the dictionary wrongly.